src/node/internal/crypto_random.ts

```diff
@@ -70,7 +72,7 @@ export function randomBytes(size: number, callback?: RandomBytesCallback) : Uint
 }
 
 export function randomFillSync(
-    buffer: ArrayBufferView|ArrayBuffer,
+    buffer: BufferLike,
```

Comment:
Is dropping `ArrayBufferView` and `ArrayBuffer` expected?

Reply:
Ah, yeah that's a good catch. I don't think the definition of `BufferLike` is complete enough here. This should accept `DataView`, `ArrayBuffer`, and `SharedArrayBuffer`

Reply:
ok, i found a better solution: `NodeJS.ArrayBufferView` defined in `@types/node`

Reply:
Hot take: We should have a linter rule to avoid calling `ArrayBufferView` and keep our internal functions types in sync with `@nodejs/types`